src/AppBundle/Controller/ArticleInfoController.php

```diff
@@ -76,9 +77,14 @@ public function resultAction(Request $request)
         $project = $request->attributes->get('project');
         $page = $request->attributes->get('article');
 
-        // sets the database name within $this->lh
-        $dbValues = $this->lh->databasePrepare($project, 'ArticleInfo');
-        $projectUrl = $dbValues['url'];
+        $projectData = ProjectRepository::getProject($project, $this->container);
+
+        if (!$projectData->exists()) {
+            $this->addFlash("notice", ["invalid-project", $project]);
+            return $this->redirectToRoute("articleInfo");
```

Comment:
Indeed.  I wonder if we can make our forms smart enough that we can post back and have the form display the values that way.  I'm not 100% sure though...